ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -177,14 +177,15 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                             Image(uiImage: trailingImage)
```

Comment:
if client passes in custom trailing image, how do they set accessibilitylabel?

Reply:
Yeah, I was going to add that today, just ran out of time yesterday.